src/layout/Panel/PanelGroupContainer.tsx

```diff
@@ -162,31 +147,25 @@ export function PanelGroupContainer({ container, components }: IPanelGroupContai
                     {body}
                   </Grid>
 
-                  {referencedGroup &&
-                    createRepeatingGroupComponentsForIndex({
-                      container: referencedGroup,
-                      renderComponents:
-                        components || referencedGroup.children.map((id) => getLayoutComponentById(id, layouts)),
-                      textResources,
-                      index: referencedGroupIndex + 1,
-                      hiddenFields,
-                    }).map((component) => {
-                      return renderLayoutComponent(component, layout);
-                    })}
-
-                  {!referencedGroup &&
-                    components.map((component) => {
-                      return renderLayoutComponent(component, layout);
-                    })}
-
-                  {referencedGroup && (
-                    <Grid item>
-                      <SuccessIconButton
-                        id={`save-reference-button-${container.id}`}
-                        label={getLanguageFromKey('general.save', language)}
-                        onClick={handleSave}
-                      />
-                    </Grid>
+                  {/*  // TODO: Add test case for filling out a new row in panel, not saving it, and midway through*/}
```

Comment:
Should we create an issue on the backlog to ensure this TODO get prioritized? 🤔 :D

Reply:
Yeah, I'll do that. My plan was to do that ASAP, as the Panel thing turned out to be horribly broken (but was just as broken before this PR, so I didn't break it). Better to create an issue in case I don't get around to it, so I'll do that.

Reply:
- #985
It got rather large in the end, when I wrote down everything I was thinking about. Good thing it's documented in an issue now, so thanks for that nudge!